sales/sales/sales.py

```diff
@@ -0,0 +1,300 @@
+                    salary=self.salary,
+                )
+            )
+            session.commit()
+        return pc.window_alert(f"User {self.customer_name} has been added.")
+
+    def customer_page(self):
+        """The customer page."""
+        return pc.redirect("/")
+    
+    def onboarding_page(self):
+        """The onboarding page."""
+        return pc.redirect("/onboarding")
+
+    def delete_customer(self, email: str):
+        """Delete a customer from the database."""
+        with pc.session() as session:
+            session.query(Customer).filter_by(email=email).delete()
+            session.commit()
+
+    def call_openai(
+        self,
+        name: str,
+        email: str,
+        age: int,
+        gender: str,
+        location: str,
+        job: str,
+        salary: int,
+    ):
+        """Call OpenAI to generate a personalized response."""
+        response = openai.Completion.create(
+            model="text-davinci-003",
+            prompt=f"Based on these {products} write a sales email to {name} adn email {email} who is {age} years old and a {gender} gender. {name} lives in {location} and works as a {job} and earns {salary} per year. Make sure the email reccomends one product only and is personalized to {name}. The company is named Pynecone its website is https://pynecone.io",
+            temperature=0.7,
+            max_tokens=2250,
+            top_p=1,
+            frequency_penalty=0,
+            presence_penalty=0,
+        )
+        self.gen_response = False
+        self.response = response.choices[0].text
+
+    def send_email(
+        self,
+        name: str,
+        email: str,
+        age: int,
+        gender: str,
+        location: str,
+        job: str,
+        salary: int,
+    ):
+        """Call call_openai and set gen_response to True for progress bar."""
+        self.gen_response = True
+        return self.call_openai(name, email, age, gender, location, job, salary)
+
+    @pc.var
+    def get_users(self) -> list[Customer]:
+        """Get all users from the database."""
+        with pc.session() as session:
+            self.users = session.query(Customer).all()
+            return self.users
+
+
+def navbar():
+    """The navbar for the top of the page."""
+    return pc.box(
+        pc.hstack(
+            pc.link(
+                pc.hstack(
+                    pc.image(src="/pinecone.ico", width="50px"),
+                    pc.heading("Pynecone | Personalized Sales", size="lg"),
+                ),
+                href="https://pynecone.io",
+            ),
+            pc.menu(
+                pc.menu_button(
+                    "Menu", bg="black", color="white", border_radius="md", px=4, py=2
+                ),
+                pc.menu_list(
+                    pc.link(
+                        pc.menu_item(
+                            pc.hstack(
+                                pc.text("Customers"), pc.icon(tag="HamburgerIcon")
```

Comment:
We updated the pc.icon syntax in the latest release, can you change this to
```python
pc.icon(tag="hamburger")
```
You can upgrade pc to v0.1.18 to make sure it works